File: GestionPov/src/main/java/com/gestionDePov/GestionPov/Model/Suivi.java
package com.gestionDePov.GestionPov.Model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Data
@Table
public class Suivi {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Long idSuivi;
    public boolean OffreCommercial;
    public float   Montant;

    //idTypePres
    @ManyToOne
    @JoinColumn(name = "idTypePrestation")
    public TypePrestation TypePres;


    public String CompteRendu;

    //idPov
    @ManyToOne
    @JoinColumn(name = "idPov")
    public POV povSuivi;

}

File: GestionAppliance/src/app/Model/suivi.ts
import { Pov } from "./pov";
import { TypePrestation } from "./type-prestation";

export interface Suivi {
    idSuivi:number;
    offreCommercial?: boolean;
    montant?: number;
    compteRendu?:String;
    typePres: TypePrestation;
    povSuivi:Pov
}
